pylint/reporters/reports_handler_mix_in.py

```diff
@@ -33,25 +37,25 @@ def register_report(self, reportid, r_title, r_cb, checker):
         reportid = reportid.upper()
         self._reports[checker].append((reportid, r_title, r_cb))
 
-    def enable_report(self, reportid):
+    def enable_report(self, reportid: str) -> None:
         """disable the report of the given id"""
         reportid = reportid.upper()
         self._reports_state[reportid] = True
 
-    def disable_report(self, reportid):
+    def disable_report(self, reportid: str) -> None:
         """disable the report of the given id"""
         reportid = reportid.upper()
         self._reports_state[reportid] = False
 
-    def report_is_enabled(self, reportid):
+    def report_is_enabled(self, reportid: str) -> bool:
         """return true if the report associated to the given identifier is
         enabled
         """
         return self._reports_state.get(reportid, True)
 
-    def make_reports(self, stats, old_stats):
+    def make_reports(self, stats: Dict[str, Any], old_stats: Dict) -> Section:
```

Comment:
We should probably block this PR by https://github.com/PyCQA/pylint/pull/4973/
This function is also typed more specifically there.